From d56f2e2f79e7dcb9265373e97d2485bf40ab8d49 Mon Sep 17 00:00:00 2001
From: Eric Le Bihan <eric.le.bihan.dev@free.fr>
Date: Tue, 16 Aug 2016 10:24:51 +0200
Subject: [PATCH] Fix spawn_lib support

Signed-off-by: Eric Le Bihan <eric.le.bihan.dev@free.fr>
---
 configure | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/configure b/configure
index ab7279f..66a76f4 100755
--- a/configure
+++ b/configure
@@ -289,7 +289,7 @@ if [ "x$target" != "x$(cat $sysdeps/target)" ] ; then
   exit 1
 fi
 
-rt_lib=$(cat $sysdeps/rt.lib)
+spawn_lib=$(cat $sysdeps/spawn.lib)
 socket_lib=$(cat $sysdeps/socket.lib)
 sysclock_lib=$(cat $sysdeps/sysclock.lib)
 tainnow_lib=$(cat $sysdeps/tainnow.lib)
@@ -365,7 +365,7 @@ sproot := $sproot
 version := $version
 home := $home
 exthome := $exthome
-RT_LIB := ${rt_lib}
+SPAWN_LIB := ${spawn_lib}
 SOCKET_LIB := ${socket_lib}
 SYSCLOCK_LIB := ${sysclock_lib}
 TAINNOW_LIB := ${tainnow_lib}
-- 
2.4.11

